end-to-end-tests/src/test/java/io/eventuate/examples/tram/sagas/ordersandcustomers/endtoendtests/CustomersAndOrdersE2ETest.java

```diff
@@ -67,21 +60,47 @@ public void shouldRejectBecauseOfInsufficientCredit() {
   @Test
   public void shouldRejectBecauseOfUnknownCustomer() {
 
-    CreateOrderResponse createOrderResponse = restTemplate.postForObject(baseUrlOrders("orders"),
+    CreateOrderResponse createOrderResponse = restTemplate.postForObject(baseUrl("orders"),
             new CreateOrderRequest(Long.MAX_VALUE, new Money("123.40")), CreateOrderResponse.class);
 
     assertOrderState(createOrderResponse.getOrderId(), OrderState.REJECTED, RejectionReason.UNKNOWN_CUSTOMER);
   }
 
+  @Test
+  public void shouldSupportHistory() {
+    CreateCustomerResponse createCustomerResponse = restTemplate.postForObject(baseUrl("customers"),
+            new CreateCustomerRequest("John", new Money("1000.00")), CreateCustomerResponse.class);
```

Comment:
There are a bunch of magic values, such as `John`, that should be refactored to constants, `CUSTOMER_NAME`.

Reply:
Ok